src/components/Dex.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <section id="dex">
+    <div class="background">
+      <img src="~@/assets/img/Blurred_Bg_Wallets.png">
+    </div>
+    <div class="header">
+      <div class="title">
+        {{ currentTab }}
+      </div>
+    </div>
+    <router-view></router-view>
+    <div class="footer">
+      <router-link to="/authenticated/dex/pair">
+        <div @click="setTab">
+          Pair
+        </div>
+      </router-link>
+      <router-link to="/authenticated/dex/trade">
+        <div @click="setTab">
+          Trade
+        </div>
+      </router-link>
+      <router-link to="/authenticated/dex/orders">
+        <div @click="setTab">
+          Orders
+        </div>
+      </router-link>
+      <router-link to="/authenticated/dex/charts">
+        <div @click="setTab">
+          Charts
+        </div>
+      </router-link>
+    </div>
+  </section>
+</template>
+
+<script>
+export default {
+  mounted() {
+    this.loadMarkets();
+  },
+
+  computed: {
+    //
+  },
+
+  data() {
+    return {
+      currentTab: 'Pair',
+    };
+  },
+
+  methods: {
+    loadMarkets() {
+      this.$store.dispatch('fetchMarkets', {
+        done: () => {
+          if (!this.$store.state.currentMarket) {
+            this.$store.commit('setCurrentMarket', this.$store.state.markets[0]);
+          }
+        },
+      });
+    },
+
+    setTab(event) {
+      this.currentTab = event.target.innerText;
```

Comment:
I know, this is hacky. I'm sure there's a better way to do this, using router, but I couldn't seem to get it to work and I decided to do this so I could move on. I'm sure the real solution is simple.

Reply:
Yes this should work with using `<router-link />` and styling `.router-link-active`.